internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -33,55 +41,68 @@ func NewServer(c config.Configuration) (*Server, error) {
 func (srv *Server) RuleContext(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		var rule *models.BounceRule
-		var err error
-		if bounce_id := chi.URLParam(r, "bounce_id"); bounce_id != "" {
-			rule, err = srv.DBClient.GetRuleDB(bounce_id)
+
+		bounce_id := chi.URLParam(r, "bounce_id"); 
+		bouncd_idInt, err := strconv.Atoi(bounce_id)
+		
+		if err != nil {
+			log.Println(err)
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		if bounce_id != "" {
```

Comment:
I don't think we need to check for empty string here, because `strconv.Atoi` should return err if `bounce_id` is empty string....